lib/foreman/renderer.rb

```diff
@@ -49,6 +50,16 @@ def host_param_false?(name)
       @host.params.has_key?(name) && Foreman::Cast.to_bool(@host.params[name]) == false
     end
 
+    def subnet_has_param?(subnet, param_name)
+      raise ::Foreman::Renderer::WrongSubnetError.new(N_("'%{object_name}' is a '%{object_class}', expected a subnet.") % { object_name: subnet.to_s, object_class: subnet.class.to_s}) unless subnet.is_a?(Subnet)
+      return subnet.parameters.exists?(name: param_name)
+    end
+
+    def subnet_param(subnet, param_name)
+      param = subnet.parameters.where(name: param_name).first
```

Comment:
this needs the same check for subnet type, otherwise I'm getting `undefined method `parameters' for "1":String` when I use it incorrectly